shaders/Membrane.gdshader

```diff
@@ -10,9 +10,10 @@ uniform sampler2D albedoTexture : source_color;
 uniform sampler2D damagedTexture : source_color;
 
 uniform sampler2D normalTexture;
+uniform sampler2D wiggleNormalTexture;
```

Comment:
Why is this called "wiggleNormal"? As far as I can see there isn't a difference between the first normal map and the second in terms of wiggle (both seem to be now affected by it). Is there a typo perhaps or did you just mean to have two normal maps? In the latter case I think the variable needs renaming (and the relevant variables in the membrane C# code).

Reply:
The idea was to have them move separately, although I'm not sure there's a need for it. I'll look at making it move relative to the other normal. The alternative would be reverting most of the shader changes and making the new normals more bumpy.

Reply:
If you didn't mean to break a fix by @dligr the normal bump map is supposed to use UV**2** and not just the first UV. So if you redo that change to preserve that, then this would all make sense as the second bump map would be affected by the wiggles and the first one wouldn't be.

Reply:
I've experimented with this, and I just don't see any difference in using the UV2. The shader was scaling the UV2 and applying the normal map on the basis of that. The other effects on the normal were based on the value of the normal map as applied, and it makes no difference if it's UV1 or UV2.
What I've now done is removed the second normal map, replaced the normal map texture with one that includes any extra bumpiness required, and applied it via UV1. The effect is the same whichever channel I use, so I might as well use the same one as the albedo.

Reply:
The way UV2 was calculated is actually not scaling, but rather something like a projection from a flat plane. UV1 is different in that it's kind of wrapped around the cell.

You can check the difference on larget cells, where bumps would be more streched, while with UV2 they would repeat.

Reply:
Okay, I see your point. I hadn't fully compared larger and smaller cells. I'll have to change it back to split normals.
What's the reason for setting the UV2 scale in vertex()?